src/components/LoadingSpinner.tsx

```diff
@@ -25,11 +25,21 @@ interface LoadingSpinnerProps {
   style?: CSSProperties
 }
 
-const LoadingSpinner: FC<LoadingSpinnerProps> = ({ size, style }) => <Spinner size={size} style={style} />
+const LoadingSpinner: FC<LoadingSpinnerProps> = ({ size, style }) => (
+  <SpinnerContainer style={{ height: size, width: size }}>
+    <Spinner style={style} />
+  </SpinnerContainer>
+)
```

Comment:
I noticed that the snackbar message gets hidden behind the spinner, not sure if it's related to this file

<img width="1728" alt="image" src="https://github.com/alephium/explorer/assets/1579899/3eaf40e1-020c-4630-b462-9e1939203de4">